mudirac/mudirac.xml

```diff
@@ -163,9 +164,21 @@ electronic_config: $electronic_config
         <param name="uehling_correction" type="boolean" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not" optional="true"/>
         <param name="electronic_config" type="text" value="" label="electronic configuration to use in order to describe the negative charge background" help="Can be a full string describing the configuration (e.g. 1s2 2s2 2p2), 
         an element symbol to represent the default configuration of that atom when neutral (e.g. C) or a mix of the two (e.g. [He] 2s2 2p2)" optional="true"/>
+        <param name="output_level" type="select" label="ouput level">
+            <option value="1" selected="true"> Only transition energies and rates</option>
+            <option value="2"> Transiion energies, states and transition matrices</option>
+        </param>
```

Comment:
Since we only have two options here, I think we could either:
- Use `display="radio"` for radio buttons rather than a drop down select box. It's not an exact science but I think general UI/UX best practice is to use radio when there are less than ~5 options.
- Alternatively, since we only have 2 options we could make this a boolean parameter to include states and matrices in the output. Could do this by setting `truevalue="2"`, `falsevalue="1"` in the xml attributes and keeping the config formatting the same.

Personally I think boolean would be the most intuitive but if you prefer the current UI we can keep it as that, just want to make sure we've considered all the possible options.